apps/website/src/icons/component-icons/Alert.astro

```diff
@@ -0,0 +1,20 @@
+<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 100 100' {...Astro.props}>
+  <g id='b'>
```

Comment:
most of these svgs have single letter ids.

`id` must be globally unique across the whole page, so this is invalid html. one option is to run all the svgs through [`svgo`](https://github.com/svg/svgo), or you could remove the ids manually for now.

Reply:
removed IDs